app/components/LazyImage/index.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ImageArt = styled.img`
+  && {
+    object-fit: contain;
+    height: 250px;
+  }
+`;
+
+const useProgressiveImg = (lowQualitySrc, highQualitySrc) => {
```

Comment:
would love to see a custom hook for this. 

You can refer to this implementation: https://github.com/wednesday-solutions/react-screentype-hook

Reply:
okay , will get on it